lblod/frontend-worship-decisions
# [DL-5330] [filter-form] Replace `Type bestuursorgaan` filter for `Type eredienst`

# Description
DL-5330

This PR replaces current filter `Type bestuursorgaan` filter for `Type eredienst` so users can look for recognized worship types when they're already filtering on `Type bestuur` leaving them 6 options to pick from _(Anglicaanse, Islamitische, Israëlitische, Orthodox, Protestantse of Rooms-Katholiek)_.

# Type of change


# Related services

- [frontend-worship-decisions](https://github.com/lblod/frontend-worship-decisions)
- mu-cl-resource
- [toezicht-search-query-management](https://github.com/lblod/toezicht-search-query-management-service)

# How to test (From Loket app)

1. drc up -d 
2. Make sure the seach-query-management and frontend services have been pulled.
a. You can always proxy the ember app instead of pulling the image if that doesn't work
3. Log as ABB.
4. Apply type bestuur filter then type eredienst based on the 6 options.
5. It should show results.

**See Examples here**

![Screenshot 2024-06-10 at 18-04-26 Erediensten Toezichtsdatabank](https://github.com/lblod/app-worship-decisions-database/assets/38471754/3326c9c4-1098-43ff-9521-04a0c32e81bf)

![Screenshot 2024-06-10 at 18-04-47 Erediensten Toezichtsdatabank](https://github.com/lblod/app-worship-decisions-database/assets/38471754/f0aa0341-9ba3-45b5-be6c-e51443a3472c)


# What to check

- Typos 

# Links to other PR's

- https://github.com/lblod/toezicht-search-query-management-service/pull/1
- https://github.com/lblod/app-worship-decisions-database/pull/81

# Notes

- Make proper release + CHANGELOG
